src/wallet/api/wallet.h

```diff
@@ -214,7 +214,9 @@ class WalletImpl : public Wallet
     virtual void startRefresh() override;
     virtual void pauseRefresh() override;
     virtual bool parse_uri(const std::string &uri, std::string &address, std::string &payment_id, uint64_t &amount, std::string &tx_description, std::string &recipient_name, std::vector<std::string> &unknown_parameters, std::string &error) override;
+    bool parse_uri(const std::string &uri, std::vector<std::string> &addresses, std::vector<std::uint64_t> &amounts, std::vector<std::string> &recipient_names, std::string &tx_description, std::vector<std::string> &unknown_parameters, std::string &error) override;
```

Comment:
```suggestion
    bool parse_uri(const std::string &uri, std::vector<std::string> &addresses, std::vector<uint64_t> &amounts, std::vector<std::string> &recipient_names, std::string &tx_description, std::vector<std::string> &unknown_parameters, std::string &error) override;
```

So that the signature matches with the existing one. Please do the same thing with `make_uri`.

Reply:
thanks for introducing the problem, surprised it went overlooked till now. The entire code had `std::uint64_t` and `uint64_t` inconsistency. I've updated the code only to use `uint64_t` :)